wbb/modules/paste.py

```diff
@@ -29,7 +29,7 @@ async def paste(client, message):
             await m.edit(final_link, disable_web_page_preview=True)
     elif bool(message.reply_to_message) is False:
         await message.reply_text(
-            '''Reply To A Message With /paste, Just Hitting /paste
+            '''```Reply To A Message With /paste, Just Hitting /paste
 Won't Do Anything Other Than Proving Everyone That
-You Are Obsessed To 'BlueTextMustClickofobia'.'''
+You Are Obsessed To 'BlueTextMustClickofobia'```'''
         )
```

Comment:
should use `+` here, multiline string adds new line character.

Reply:
should i add '+' with single line strings and remove multiline string?

Reply:
Fixed

Reply:
yes